scripts/gui/thrive_style.css

```diff
@@ -104,14 +115,20 @@ div::-webkit-scrollbar-thumb:hover {
   overflow: hidden;
 }
 
+
 /* Bars */
+
```

Comment:
Where did all of these blank lines come from? I'm sure stylelint doesn't enforce these on my computer or on jenkins or circleci.

Reply:
actually i thought i was using the same rules file, i go check if Atom had override some rules itself